README.md

```diff
@@ -1,5 +1,15 @@
-# User Review Question Answering System
-This repository explores the application of XL-Net on user review based question answering system. The base model and algorithm was inspired and based upon the [renatoviolin/xlnet link](https://github.com/renatoviolin/xlnet) repo.
+## [Project Demo Slides](https://docs.google.com/presentation/d/16pl_ZvUmtmWsFKmMWbTw3GtJ1R5X2A84t0INaWZ02Ek/edit#slide=id.g63c4d69c00_0_222)
+
+# OnPoint: A Question Answering Service leveraging user reviews
+![image of pipline](https://github.com/hairong-wang/XLNet_learn2learn/blob/dev-tpu_version-20191012/src/com/insightdatascience/xlnet_learn2learn/static/img/pipeline.png)
+
+OnPoint is a question answering service which levearages product user reviews. OnPoint saves you lots of time when you try to look for a product detail by providing you a short answer in seconds. 
```

Comment:
Is it true that it will take several seconds to get the answer back? If it shorter than one second then you may consider changing "...by providing you a short answer in seconds. " to "by providing you a short answer in less than one second."

Reply:
It takes several seconds.